coderd/database/migrations/000124_validation_min_max_nullable.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+ALTER TABLE template_version_parameters ALTER COLUMN validation_min DROP NOT NULL;
+ALTER TABLE template_version_parameters ALTER COLUMN validation_max DROP NOT NULL;
+UPDATE template_version_parameters SET validation_min = NULL WHERE validation_min = 0;
```

Comment:
Thinking out loud, what if someone has explicitly specified min to be 0 so that -1 can't be used? This seems like a potentially lossy migration and we should probably wait until templates are updated to receive their new, null value?